diff -uNr build.orig/config.sh build/config.sh
--- build.orig/config.sh	Tue Dec 14 03:22:20 1999
+++ build/config.sh	Fri Dec 24 16:28:44 1999
@@ -348,11 +348,7 @@
 	if [ "$INPUT" = "" ] ; then
 		INPUT=$DEFAULT_BIN_DIR
 	fi
-	if [ -d "$INPUT" ] ; then
-		BIN_DIR="$INPUT"
-	else
-		echo "\"$INPUT\" not found."
-	fi
+	BIN_DIR="$INPUT"
 done
 echo ""
 
@@ -365,11 +361,7 @@
 	if [ "$INPUT" = "" ] ; then
 		INPUT=$DEFAULT_LIB_DIR
 	fi
-	if [ -d "$INPUT" ] ; then
-		LIB_DIR="$INPUT"
-	else
-		echo "\"$INPUT\" not found."
-	fi
+	LIB_DIR="$INPUT"
 done
 echo ""
 JNAMES_DB="$LIB_DIR/jnames"
@@ -389,11 +381,7 @@
 	if [ "$INPUT" = "" ] ; then
 		INPUT=$DEFAULT_MAN_DIR
 	fi
-	if [ -d "$INPUT" ] ; then
-		MAN_DIR="$INPUT"
-	else
-		echo "\"$INPUT\" not found."
-	fi
+	MAN_DIR="$INPUT"
 done
 echo ""
 
@@ -406,11 +394,7 @@
 	if [ "$INPUT" = "" ] ; then
 		INPUT=$DEFAULT_TMP_DIR
 	fi
-	if [ -d "$INPUT" ] ; then
-		TMP_DIR="$INPUT"
-	else
-		echo "\"$INPUT\" not found."
-	fi
+	TMP_DIR="$INPUT"
 done
 echo ""
 
@@ -465,15 +449,7 @@
 	if [ "$INPUT" = "" ] ; then
 		INPUT="$DEFAULT_NEWS_SPOOL"
 	fi
-	if [ -d "$INPUT" ] ; then
-		NEWS_SPOOL="$INPUT"
-	else
-		echo "\"$INPUT\" not found."
-		Y_or_n "Ignore this error?"
-		if [ $? -ne 0 ] ; then
-			NEWS_SPOOL="$INPUT"
-		fi
-	fi
+	NEWS_SPOOL="$INPUT"
 done
 echo ""
 
@@ -486,15 +462,7 @@
 	if [ "$INPUT" = "" ] ; then
 		INPUT="$DEFAULT_NEWS_LIB"
 	fi
-	if [ -d "$INPUT" ] ; then
-		NEWS_LIB="$INPUT"
-	else
-		echo "\"$INPUT\" not found."
-		Y_or_n "Ignore this error?"
-		if [ $? -ne 0 ] ; then
-			NEWS_LIB="$INPUT"
-		fi
-	fi
+	NEWS_LIB="$INPUT"
 done
 echo ""
 
@@ -507,11 +475,7 @@
 	if [ "$INPUT" = "" ] ; then
 		INPUT="$DEFAULT_MAIL_SPOOL"
 	fi
-	if [ -d "$INPUT" ] ; then
-		MAIL_SPOOL="$INPUT"
-	else
-		echo "\"$INPUT\" not found."
-	fi
+	MAIL_SPOOL="$INPUT"
 done
 echo ""
 
@@ -529,16 +493,7 @@
 		if [ "$INPUT" = "" ] ; then
 			INPUT="$DEFAULT_POSTNEWS"
 		fi
-		if [ -f `echo "$INPUT" | $AWK '{print $1}'` ] ; then
-			POSTNEWS="$INPUT"
-		else
-			echo "Warning:\"`echo "$INPUT" | $AWK '{print $1}'`\" not found."
-			POSTNEWS="$INPUT"
-			Y_or_n "Ignore this error?"
-			if [ $? -ne 1 ] ; then
-				POSTNEWS=""
-			fi
-		fi
+		POSTNEWS="$INPUT"
 	done
 fi
 echo ""
@@ -557,11 +512,7 @@
 		if [ "$INPUT" = "" ] ; then
 			INPUT="$DEFAULT_SENDMAIL"
 		fi
-		if [ -f `echo "$INPUT" | $AWK '{print $1}'` ] ; then
-			SENDMAIL="$INPUT"
-		else
-			echo "\"`echo "$INPUT" | $AWK '{print $1}'`\" not found."
-		fi
+		SENDMAIL="$INPUT"
 	done
 fi
 echo ""
@@ -597,11 +548,7 @@
 			if [ "$INPUT" = "" ] ; then
 				INPUT="$DEFAULT_MH_PATH"
 			fi
-			if [ -d "$INPUT" ] ; then
-				MH_PATH="$INPUT"
-			else
-				echo "\"$INPUT\" not found."
-			fi
+			MH_PATH="$INPUT"
 		done
 	fi
 fi
@@ -632,16 +579,7 @@
 		if [ "$INPUT" = "" ] ; then
 			INPUT="$DEFAULT_BOARD_SPOOL"
 		fi
-		if [ -d "$INPUT" ] ; then
-			BOARD_SPOOL="$INPUT"
-		else
-			echo "\"$INPUT\" not found."
-			BOARD_SPOOL="$INPUT"
-			Y_or_n "Ignore this error?"
-			if [ $? -ne 1 ] ; then
-				BOARD_SPOOL=""
-			fi
-		fi
+		BOARD_SPOOL="$INPUT"
 	done
 fi
 echo ""
